browser/src/VariantList/Variants.tsx

```diff
@@ -193,11 +206,43 @@ const Variants = ({
     if (index === -1) {
       index = renderedVariants.length - 1
     }
-
     // @ts-expect-error TS(2531) FIXME: Object is possibly 'null'.
     table.current.scrollToDataRow(index)
   }, [positionLastClicked]) // eslint-disable-line react-hooks/exhaustive-deps
 
+  function getFirstIndexFromSearchText(
+    searchFilter: VariantFilterState,
+    variantSearched: Variant[],
+    variantsTableColumns: any
+  ) {
+    const searchedVariants = getFilteredVariants(
+      searchFilter,
+      variantSearched,
+      variantsTableColumns
+    )
+
+    if (searchedVariants.length > 0) {
+      const firstVariant = searchedVariants[0]
+      const firstIndex = renderedVariants.findIndex((variant: Variant) => variant.pos === firstVariant.pos)
+      if (positionLastClicked !== null && firstVariant.pos < positionLastClicked) {
+        return firstIndex - 10
+      }
+      return firstIndex + 10
+    }
+    return 0
+  }
+
+  const searchIndex = getFirstIndexFromSearchText(filter, renderedVariants, renderedTableColumns)
```

Comment:
Upon re-review, I noticed that due to logic that exists to re-render the variant bubble type track and the grid based on the state of the mouse in the other (e.g. highlight bubbles in the above track when a user hovers a variant in the table below), since this computation is happening outside of a scoped `useEffect` hook, this function is called anytime the mouse moves inside the variant table (potentially dozens of times per second). This was unexpected, as most components don't re-render based solely on mouse position.

While this isn't a particularly heavyweight function, I believe it is generally better practice to keep some piece of computation like this, that is only needed in a particular circumstance, happening close to where it is needed (in this case, in the `useEffect` hook below), particularly since this is computation happening in the users' browsers.